dashboard/modules/reporter/tests/test_reporter.py

```diff
@@ -98,7 +98,7 @@ def _check_workers():
                 return True
             return False
 
-    wait_for_condition(_check_workers, timeout=10)
```

Comment:
Hi @Kiko-Aumond, just a question out of my curiosity, so this increased time out would fix this issue, correct?